morpheushelper/cogs/mod.py

```diff
@@ -406,11 +406,12 @@ async def unmute(self, ctx: Context, user: Union[Member, User, int], *, reason:
     @commands.command()
     @Permission.kick.check
     @guild_only()
-    async def kick(self, ctx: Context, member: Member, *, reason: str):
+    async def kick(self, ctx: Context, member: Union[Member, int], *, reason: str):
```

Comment:
Please add `User` to the member type Union so that also Users that left the server can be used for the command.